src/Client/ClientBaseHelpers.cpp

```diff
@@ -6,6 +6,9 @@
 #include <Parsers/Lexer.h>
 #include <Common/UTF8Helpers.h>
 
+#if USE_REPLXX
+extern bool replxx_last_is_delimiter;
```

Comment:
Ok.

Communication with global variable looks dirty, at least a comment needed.
Maybe we can do it in some other way, like exposing functions in ReplxxLineReader to set this variable?

Reply:
Done.